coffee-mode.el

```diff
@@ -1208,6 +1240,9 @@ comments such as the following:
   ;; code for syntax highlighting
   (setq font-lock-defaults '((coffee-font-lock-keywords)))
 
+  ;; fix comment filling function
+  (setq-local comment-line-break-function #'coffee-comment-line-break-fn)
+  (setq-local auto-fill-function #'coffee-auto-fill-fn)
```

Comment:
Please use `set` and `make-local-variable` because coffee-mode supports < Emacs 24.3.